packages/koa-metrics/src/timing.ts

```diff
@@ -0,0 +1,25 @@
+import {StatsD} from 'hot-shots';
+import {Context} from 'koa';
+
+export function initTimer(client: StatsD, ctx: Context) {
```

Comment:
Can you explain how this would be used for timing other than the full response time? Would it be?